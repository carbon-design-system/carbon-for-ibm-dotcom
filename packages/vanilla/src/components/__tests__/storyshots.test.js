import initStoryshots from '@storybook/addon-storyshots';
import path from 'path';

/**
 * This will initialize storyshots for snapshot testing.
 * NOTE: If a molecule/organism requires deeper testing, a __tests__ folder must be created in the corresponding folder
 * and create additional tests in there.
 */
initStoryshots({
  configPath: path.resolve(__dirname, '../../../.storybook'),
  framework: 'html',
});
