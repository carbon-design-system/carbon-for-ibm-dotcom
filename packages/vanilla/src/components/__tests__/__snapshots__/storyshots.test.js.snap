// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots HorizontalRule Default 1`] = `

  
`;
